api/database/seeders/JobPosterTemplateSeeder.data.json

```diff
@@ -0,0 +1,1699 @@
+          "fr": "En tant que chef d’équipe en développement d'applications IT-03, vous dirigerez une équipe de développeuses et développeurs et d’analystes dans l’exécution de projets de développement d'applications du début à la fin. Votre rôle consiste à planifier les projets, à planifier les ressources et à évaluer les risques pour assurer la réussite de la réalisation du projet. Vous favoriserez un environnement de collaboration, encouragerez l’innovation et favoriserez l’amélioration continue des processus de développement. En tant que principale agente ou principal agent de liaison entre les équipes techniques et les parties prenantes, vous superviserez les échéanciers du projet, l’affectation du budget et les communications avec les clients, en veillant à ce qu’ils correspondent aux buts et aux objectifs opérationnels."
+        },
+        "tasks": {
+          "en": "<ul><li><p>Supervises and leads project teams in the development, integration, release, and maintenance of software solutions.</p></li><li><p>Identifies and delegates system issues to developers to debug, resolve, and produce technical documentation.</p></li><li><p>Conducts business analysis and feasibility studies, producing technical reports and recommending solutions for service enhancements.</p></li><li><p>Reviews team members' code submissions and provides feedback to ensure best practices and high performance.</p></li><li><p>Provides technical advice and guidance on application development processes and standards.</p></li><li><p>Represents the team in meetings with stakeholders to define service requirements.</p></li><li><p>Delegates tasks and supports the professional development of junior staff. </p></li></ul>",
+          "fr": "<ul><li><p>Superviser et diriger des équipes de projet dans le cadre du développement, de l’intégration, de la diffusion et de la maintenance de solutions logicielles.</p></li><li><p>Identifier et déléguer les problèmes du système aux développeurs pour déboguer, résoudre et produire la documentation technique.</p></li><li><p>Effectuer des analyses opérationnelles et des études de faisabilité, produire des rapports techniques et recommander des solutions pour l’amélioration des services.</p></li><li><p>Passer en revue le code élaboré par d’autres members de l'équipe et fournir une rétroaction pour assurer des pratiques exemplaires et un rendement élevé.</p></li><li><p>Fournir des conseils techniques et des directives sur les normes et les processus de développement des applications.</p></li><li><p>Représenter l'équipe lors des réunions avec les parties prenantes pour définir les exigences de service.</p></li><li><p>Déléguer des tâches et soutenir le développement professionnel du personnel subalterne. </li></ul>"
```

Comment:
I just noticed that this list is missing the closing `</p>`.  Are there more?

Reply:
Added what I found in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11523/commits/e2cb3fe03066d1abaa047ccfb61da22f7597f227

Reply:
I found one more on L277